composer/trainer/trainer.py

```diff
@@ -1032,8 +1101,7 @@ def _train_batch_inner(self, microbatches: Sequence[Batch]):
         current_batch_size = sum([self._train_data_spec.get_num_samples_in_batch(batch) for batch in microbatches])
 
         for microbatch_idx, self.state.batch in enumerate(microbatches):
-            self.state.batch_num_tokens = self._train_data_spec.get_num_tokens_in_batch(self.state.batch)
-            self.state.batch_num_samples = self._train_data_spec.get_num_samples_in_batch(self.state.batch)
```

Comment:
@ravi-mosaicml is it fine to get rid of these state updates?

Reply:
Will leave unresolved for now, but I cleared with him ahead of time. I think the idea was to expose microbatch size for algorithms in the future, but the naming makes it unclear what it is and causes problems with grad_accum so Ravi was fine with removing